freqtrade/exchange/exchange.py

```diff
@@ -269,6 +269,13 @@ def validate_timeframes(self, timeframe: List[str]) -> None:
         """
         Checks if ticker interval from config is a supported timeframe on the exchange
         """
+        logger.warning("validate_timerames")
```

Comment:
should be something like
```
logger.info("Validating timeframes...")
```

logger.warning() is not proper here since it's printed before checking for attr an generating the exception. A warning or even a critical message may additionally be issued below, under the `if not hasattr()` part.

Reply:
This ... was never intendet to be commited - forgot to remove it.